src/cli/lint/autofix.ts

```diff
@@ -18,6 +19,8 @@ import { REFRESHABLE_IGNORE_FILES } from '../configure/refreshIgnoreFiles';
 
 import type { Input } from './types';
 
+const RENOVATE_DEFAULT_PREFIX = 'renovate-skuba-';
```

Comment:
Can we support a few other paths for completeness:

```
renovate-all-dependencies
renovate/all-dependencies
renovate/skuba-*
```

Alternatively, is there a downside to generalising to `renovate/` and `renovate-`?

Reply:
Hmmmm lets just generalise to starts with `renovate`

Reply:
I think all renovate branches makes sense. We don't want other renovate branches getting stuck either.